src/Kiota.Builder/Writers/CSharp/CodeMethodWriter.cs

```diff
@@ -8,13 +8,9 @@
 namespace Kiota.Builder.Writers.CSharp;
 public class CodeMethodWriter : BaseElementWriter<CodeMethod, CSharpConventionService>
 {
-    /// <summary>
-    /// Flag to avoid pushing a breaking change, remove it and remove the condition that depends on it with V2
-    /// </summary>
-    private readonly bool IsRequestConfigurationGeneric;
```

Comment:
The method prototypes for the `ToXXXRequestInformation` and `GetAsync` methods have the parameter types changed from non-generic to generic requestConfiguration parameters. 
Should we generate deprecated overloads for them or keep the non-generic versions as the API signature is changed?

Reply:
We're still generating the RequestConfiguration class as `WhateverName : RequestConfiguration<QueryParameters>`
`Action<WhateverName>` should be able to be passed to `Action<RequestConfiguration<QueryParameter>`.
Unless I missed something?
And even then, the only scenario where this would be a problem is if someone is providing the callback from a property, variable or code parameter coming from the application code, if they are using it inline like I expect more people are doing, it's going to be transparent.

Reply:
> if they are using it inline like I expect more people are doing, it's going to be transparent.

+1